data/assets/examples/pointclouds/data/textured_csv/textured_points.csv

```diff
@@ -0,0 +1,6 @@
+# The texture mapping from index to file is handled in another file
+x,y,z,a,b,texture
```

Comment:
Sure. Now that it is possible to have comments at the top of a csv file, I can add it to the other datasets as well

Reply:
However, I kind of think that adding too much information to these files is a bit redundant.  I don't expect the user to look at these data files in detail

The unit and origin do not matter, since the dataset can be moved around by changing the parent node, and the unit can be changed in the asset.  It is really only dummy values to be able to render some points. But I can add a comment at the top that describes this

Reply:
Added a comment to the top of the file